docs/core/rid-catalog.md

```diff
@@ -108,8 +110,15 @@ See [Prerequisites for .NET Core on Windows](windows-prerequisites.md) for more
 
 ## Linux RIDs
 
-- Portable
+Only common values are listed. For the latest and complete version, see the [runtime.json](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) file on CoreFX repo. Devices running a distribution not listed below may work with one of the Portable RIDs. For example, Raspberry Pi devices running a Linux distribution not listed can be targeted with `linux-arm`.
+
+- Portable (.NET Core 2.0 or later versions)
+  - `linux-arm`
+  - `linux-musl`
```

Comment:
This should be more like `linux-musl-x64`. It may be a good idea to have a dedicated top-level bullet point for musl-based Linux distros.

Reply:
"It may be a good idea to have a dedicated top-level bullet point for musl-based Linux distros."
If I knew more about that I would happily add it but I'm not super familiar with that area. I'm interpreting the other stated change as leaving linux-arm (and linux-x64) and changing linux-musl to linux-musl-x64